.gitignore

```diff
@@ -0,0 +1 @@
+test.S
```

Comment:
I have learned to like the "build"-directory style of things.

So mid-term, I would suggest to put build-time artefacts in some `build/` directory and only add that to gitignore.

Reply:
Yes. As long as it's one generated file vs. one generated directory. I'm not ready to add complexity. Keep it simple for now.